README.md

```diff
@@ -1,152 +1,187 @@
+### Dependencies
 
-## Getting Started
+The following are dependencies for building and running SNEK:
 
-### Dependencies and Pre-Requisites
-
-Before building the respository, you'll need the following:
-
-1. [CMAKE](https://cmake.org/) - required for building `glfw`.
-2. **WINDOWS ONLY**: This project relies primarily on Makefiles for compilation. Our tool of choice for compiling cpp on Windows is [Mingw64](http://mingw-w64.org/doku.php), since it's fully compatible with other operating system compilation methods. A comprehensive guide for installing Mingw64 can be found [here](https://www.youtube.com/watch?v=aXF4A5UeSeM). Once installed, you should be able to pull in your compiler of choice (we recommend gcc or clang).  
-3. Vulkan - the SDK can be downloaded from the [LunarG website](https://vulkan.lunarg.com/). 
-    
-    3.1. Once the Vulkan SDK is installed, you'll want to make sure that both the `VULKAN_SDK` and `VK_VERSION` environment variables are set on your system. Platform specific instructions can be found below:
-    
-    *  **macOS & Linux:**
-        Both MacOS and Linux have a similar process for setting their environment variables. The only difference between the two is their SDK directory structure. The two directory structures have been included below:
-        ```bash
-        # For macOS
-        $ export VULKAN_SDK=<VULKAN_INSTALL_DIR>/VulkanSDK/<VERSION>/macOS
-        
-        # for Linux
-        $ export VULKAN_SDK=<VULKAN_INSTALL_DIR>/VulkanSDK/<VERSION>/x86_64
-        
-        # for both
-        $ export VK_VERSION=<VERSION>
-        ```
-        Remember to substitute `<VULKAN_INSTALL_DIR>` with the directory Vulkan was installed into, and `<VERSION>` with your Vulkan version, i.e: 1.2.176.1.
-    * **Windows:** 
-        For Windows, the SDK should be automatically installed to your `C:` directory. To set these variables, you can set the following:
-        ```shell
-        > setx VULKAN_SDK = <VULKAN_INSTALL_DIR>/VulkanSDK/<VERSION>
-        > setx VK_VERSION = <VERSION>
-        ```
-        For more information about this syntax, please check the following [guide](https://www.shellhacks.com/windows-set-environment-variable-cmd-powershell/). Environment variables can also be set manually using Windows system properties. 
-        
-        A guide on this process can be found [here](https://www.alphr.com/environment-variables-windows-10/).
-
-### Building the project
-Once you have cloned this repository and installed dependencies, building the project is as simple as running these two commands in its root directory:
-
-#### macOS & Linux
-```bash
-$ make setup
-$ make
-```
+- [Cmake](https://cmake.org/) (all platforms)
+- [GNU Make](https://www.gnu.org/software/make/) (linux & macos)
+- [MingW32-make](https://www.mingw-w64.org/) (windows)
+- [Python](https://www.python.org/) (all platforms)
```

Comment:
what requires Python?

Reply:
The Validation layers has a dependency that uses a Python script. Unfortunately I haven't found a way to get around this limitation just yet.